code/game/jobs/job/civilian.dm

```diff
@@ -291,8 +292,10 @@
 		H.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses/big(H), slot_glasses)
 		H.equip_to_slot_or_del(new /obj/item/device/pda/lawyer(H), slot_belt)
 		H.equip_to_slot_or_del(new /obj/item/weapon/storage/briefcase(H), slot_l_hand)
+		H.equip_to_slot_or_del(new /obj/item/weapon/melee/telebaton(H), slot_in_backpack) //remove those when the event is over
+		H.equip_to_slot_or_del(new /obj/item/weapon/gun/projectile/colt(H), slot_in_backpack) //remove those when the event is over
 
-		H.implant_loyalty(H)
+		H.implant_loyalty_sol(H)

```

Comment:
Is there any way we can check this guy's job preference and choose the implant based on that?